### PR TITLE
Some new rules

### PR DESCRIPTION
**Pull request by Haron-Prime**
Add thunderbird.rules, shutter.rules, libreoffice.rules, FoxitReader.rules, i3wm.rules, awesome_wm.rules.
Update xmonad.rules

### CODE REVIEW

File: ananicy.d/00-default/thunderbird.rules [hunk 1]
```diff
@@ -0,0 +1 @@
+NAME=thunderbird TYPE=BG_CPUIO
```
Comment (Nefelim4ag) [inline ananicy.d/00-default/thunderbird.rules]: @Haron-Prime, everyone know that thunderbird is, but please add link :)

**Comment by Haron-Prime** [comment]
Sorry.!
Fixed

**Comment by Haron-Prime** [comment]
I checked the presence of this problem - this is observed in some cases.
Some running applications inherit the priority of the window manager.
Unfortunately, I could not find an acceptable solution. I think that the rules for awesome wm, i3wm and xmonad should be removed.
If I can find a solution that will allow me to use these rules in the future, I will inform you.

**Comment by Haron-Prime** [comment]
All the necessary changes, as well as two new rules for the players, are prilogayu in the next pull-request.
Closed.

**Pull request merged**